h/services/annotation_metadata.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+from sqlalchemy.dialects.postgresql import insert
+from sqlalchemy.orm import Session
+
+from h.models import Annotation, AnnotationMetadata, User
+from h.security import decrypt_jwe_dict
+
+
+class AnnotationMetadataService:
+    def __init__(self, db: Session):
+        self._db = db
+        self._secret = os.environ.get("JWE_SECRET_LMS")
```

Comment:
I had little dict with different secrets per authority but scrapped it.

We only use this now in LMS, we'll see how/when we use this more generally.